packages/connectivity/lib/connectivity.dart

```diff
@@ -93,6 +93,10 @@ class Connectivity {
   Future<String> getWifiIP() async {
     return await methodChannel.invokeMethod<String>('wifiIPAddress');
   }
+  
+  Future<String> getMobileConnectionType() async {
```

Comment:
missing dart doc.

Reply:
dart doc is still missing.
See https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#documentation-dartdocs-javadocs-etc for more details please.